torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -394,24 +461,42 @@ void unpackQuantizedWeightsHelper(
         input_val->type()->str());
 
     auto input_node = match_vmap.at(vmap.at("r"))->node()->inputs()[0]->node();
-    auto input_scale = getScaleFromInput(input_node);
-    auto q_bias = at::quantize_per_tensor(
-        original_bias, weight_scale * input_scale, 0, at::kQInt32);
-
-    std::vector<int64_t> bias_values;
-    bias_values.reserve(q_bias.numel());
-    auto bias_data = (int32_t*)q_bias.data_ptr<c10::qint32>();
-    for (const auto i : c10::irange(q_bias.numel())) {
-      bias_values.push_back(bias_data[i]);
+    at::Tensor q_bias;
+
+    if (operator_export_type == ::torch::onnx::OperatorExportTypes::ONNX_ATEN_FALLBACK) {
+      auto input_scale = getScaleFromInput(input_node);
+      q_bias = at::quantize_per_tensor(original_bias, weight_scale * input_scale, 0, at::kQInt32);
+      std::vector<int64_t> bias_values;
+      bias_values.reserve(q_bias.numel());
+      auto bias_data = (int32_t*)q_bias.data_ptr<c10::qint32>();
+      for (const auto i : c10::irange(q_bias.numel())) {
+        bias_values.push_back(bias_data[i]);
+      }
+      Node* c2_bias = CreateQuantizedBiasCaffe2(
+          bias_values,
+          graph,
+          q_bias.sizes().vec(),
+          q_bias.q_scale(),
+          q_bias.q_zero_point());
+      c2_bias->insertBefore(qlinear_node);
+      qlinear_node->insertInput(2, c2_bias->output());
+    } else if (operator_export_type == ::torch::onnx::OperatorExportTypes::ONNX) {
+      q_bias = at::quantize_per_tensor(original_bias, 1.0, 0, at::kQInt32);
+      std::vector<float> bias_values;
```

Comment:
Why float and not int32 or int64?
Please add a comment in the code.

Reply:
Just did some refactoring here, we need a float here because the `original_bias` is actually a float. Should not have int32 or int64 (that is used for caffe2 case only).